lib/OnyxUtils.ts

```diff
@@ -939,53 +854,40 @@ function getCollectionDataAndSendAsObject(matchingKeys, mapping) {
  *
  * @example
  * scheduleSubscriberUpdate(key, value, subscriber => subscriber.initWithStoredValues === false)
- *
- * @param {String} key
- * @param {*} value
- * @param {*} prevValue
- * @param {Function} [canUpdateSubscriber] only subscribers that pass this truth test will be updated
- * @returns {Promise}
  */
-function scheduleSubscriberUpdate(key, value, prevValue, canUpdateSubscriber = () => true) {
+function scheduleSubscriberUpdate<TKey extends OnyxKey>(
+    key: TKey,
+    value: OnyxEntry<OnyxValue<TKey>>,
```

Comment:
```suggestion
    value: OnyxValue<TKey>,
```

Reply:
This causes errors when calling this function